components/widgets/menu/menu.jsx

```diff
@@ -58,17 +66,20 @@ export default class Menu extends React.PureComponent {
                 active={this.state.focusTrapped}
                 focusTrapOptions={{
                     clickOutsideDeactivates: true,
+                    initialFocus: this.focusContainer,
                 }}
             >
                 <div>
                     <ul
                         id={id}
+                        tabIndex='-1'
                         onClick={this.removeFocusTrap}
                         className='a11y__popup Menu dropdown-menu'
                         ref={this.node}
                         style={styles}
                         role='menu'
                         aria-label={ariaLabel}
+                        onMouseOver={this.removeFocus}
```

Comment:
i think a better name for this would be `this.focus`, as the function `removeFocus` is actually invoking `.current.focus()`

Reply:
It is removing focus from the elements (that is the default behaviour), and adding it to the container, so visually it actually is removing focus from the elements.

So I'll just say `focusContainer`.

Reply:
Cool.